<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="FirstStack">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>891</width>
      <height>621</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>291</width>
      <height>211</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>891</width>
      <height>621</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="MainButtons">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>prayer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
     <widget class="QLabel" name="DateTime">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Digital_Clock">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>161</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Weather</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>81</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Bluetooth</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>130</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>280</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Camera">
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>201</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Camera</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>201</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Camera</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <zorder>pushButton_4</zorder>
     <zorder>widget</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>pushButton_5</zorder>
    </widget>
    <widget class="QWidget" name="PrayerTimes">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>161</width>
        <height>18</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>161</width>
        <height>18</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>191</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>340</y>
        <width>251</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Weather"/>
    <widget class="QWidget" name="Settings">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>194</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Time:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>340</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply changes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Bluetooth">
     <widget class="QLabel" name="bluetooth_icon">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>161</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="bluetooth_state">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>211</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="toggle_bluetooth">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>400</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>home</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
